presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraMetadata.java

```diff
@@ -169,7 +170,7 @@ private ConnectorTableMetadata getTableMetadata(ConnectorSession session, Schema
         ImmutableMap.Builder<SchemaTableName, List<ColumnMetadata>> columns = ImmutableMap.builder();
         for (SchemaTableName tableName : listTables(session, prefix)) {
             try {
-                columns.put(tableName, getTableMetadata(session, tableName).getColumns());
+                columns.put(tableName, getTableMetadata(tableName).getColumns());
```

Comment:
similar to the other list* methods, Presto may expect to be getting lower cased output from it.